cli/cmd/deploy/deploy.go

```diff
@@ -57,6 +58,14 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				}
 			}
 
+			// telemetry errors shouldn't fail deploy command
+			tel, _ := telemetry.NewTelemetry(cfg.Version)
+			tel.EmitDeployStart()
```

Comment:
Check `err` or calls on `tel` will have nil pointer panics

Reply:
Will just disable the telemetry if there are any errors. Otherwise there will be too much boilerplate everywhere `tel.<method>()` calls are made